MinVer/MinVer.csproj

```diff
@@ -9,26 +9,21 @@
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <!-- NU5100: The folder being packaged contains an assembly file that is not in lib folder. -->
     <NoWarn>$(NoWarn);NU5100</NoWarn>
-    <OutputType>Exe</OutputType>
     <PackageIcon>minver.png</PackageIcon>
     <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
     <PackageProjectUrl>https://github.com/adamralph/minver</PackageProjectUrl>
     <PackageReadmeFile>README.md</PackageReadmeFile>
     <PackageReleaseNotes>https://github.com/adamralph/minver/blob/main/CHANGELOG.md</PackageReleaseNotes>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
-    <RollForward>major</RollForward>
     <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
-    <TargetFrameworks>net6.0;net7.0;net8.0</TargetFrameworks>
+    <TargetFrameworks>net6.0</TargetFrameworks>
     <LangVersion>default</LangVersion>
+    <Version>6.0.0</Version>

```

Comment:
@bording why is this here?

Reply:
I just wanted to increase the version in my spike to make it easier to try out.